lib/active_hash/base.rb

```diff
@@ -57,7 +57,11 @@ def normalize(v)
     end
 
     if Object.const_defined?(:ActiveModel)
-      extend ActiveModel::Naming
+      if Object.const_defined?(:I18n)
```

Comment:
[`activemodel` depends on `activesupport`](https://rubygems.org/gems/activemodel), also, [`activesupport` depends on `i18n`](https://rubygems.org/gems/activesupport).  So It looks doesn't need.
Have you ever encountered with situation that `activemodel` exists but not `i18n` ?

Reply:
You are right, the line seems to be redundant, and I also cannot think of a situation where `ActiveModel` is defined buy I18n is not available..

I'll make it simpler.